tests/testthat/test-renderer5-ChromHMMiterations.R

```diff
@@ -67,5 +67,5 @@ test_that("fill not constant in probability legend and circles", {
 test_that("tile stroke is black", {
   stroke.vec <- getStyleValue(
     info$html, '//g[@class="geom1_tile_parameters"]//rect', "stroke")
-  expect_true(all(stroke.vec=="black"))
+  expect_color(stroke.vec, "black")
```

Comment:
hi @Faye-yufan this should fix the unexpected test failure we observed today. the text returned by getStyleValue is actually not "black" but some RGB representation, which is computed inside the new `expect_color` helper function, which you should use for your color_off tests as well.

Reply:
yes, thanks so much!